CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [KrauseFx](https://github.com/KrauseFx)
   [#7250](https://github.com/CocoaPods/CocoaPods/pull/7250)
 
+* Update validator to stream output as xcodebuild runs  
```

Comment:
this will need to be moved to the correct section

Reply:
I updated the changelog, but what changed in the patch you gave? I see you're printing the exception where it's caught, but I'm not sure what the line that starts `p [` does. Is it logging the command arguments before calling into `Executable.execute_command`?

I also noticed a typo with "xcodebiuld" instead of "xcodebuild". Is that the only section that's different than the way I wrote it?

Reply:
Also, I rebased against master, and it looks like it broke the build

Reply:
you need to return the exceptions message in `xcodebuild`, otherwise you get the backtrace issue that dimitris commented about

Reply:
@segiddins I pushed an updated commit incorporating the `e.message` line and `p [:xcodebuild, *command, raise_on_failure]`

Reply:
@segiddins Were you able to take a look at why the CI is failing? I'm not able to tell.

Reply:
@segiddins It looks like the `xcodebuild with correct arguments when skipping import validation` spec is failing on the 4 lines that check `xcodebuild` output. Can you please help me get this out the door? Once the spec passes, I'll sync with upstream changes again.